zerver/tests/test_slack_importer.py

```diff
@@ -411,16 +411,18 @@ def test_channel_message_to_zerver_message(self, mock_build_zerver_usermessage:
         zerver_subscription = []  # type: List[Dict[str, Any]]
         added_channels = {'random': ('c5', 1), 'general': ('c6', 2)}  # type: Dict[str, Tuple[str, int]]
         zerver_message, zerver_usermessage, attachment, uploads, \
-            reaction = channel_message_to_zerver_message(1, user_data, added_users, added_recipient,
-                                                         all_messages, zerver_subscription, [],
-                                                         added_channels, 'domain')
+            reaction, id_list = channel_message_to_zerver_message(
+                1, user_data, added_users, added_recipient,
+                all_messages, zerver_subscription, [],
+                added_channels, (0, 0, 0, 0), 'domain')
         # functioning already tested in helper function
         self.assertEqual(zerver_usermessage, [])
         # subtype: channel_join is filtered
         self.assertEqual(len(zerver_message), 5)
 
         self.assertEqual(uploads, [])
         self.assertEqual(attachment, [])
+        self.assertEqual(id_list, (5, 2, 1, 0))
 
```

Comment:
To ensure the chunking works reliably, and demonstrates the `id_list` usage, you could manually chunk this data and test the outcomes?

Reply:
I am doing this in6abf87fdd0b74b2efdf5ca1d1e52b039a515e627 in the tests. I have take `chunk_size` as 1 and testing the outcomes.

Reply:
Right, I was looking for it in an earlier commit, since that is where you added the feature to the function.